middleware-jackson/src/main/java/com/nytimes/android/external/store3/middleware/jackson/JacksonStringParser.kt

```diff
@@ -0,0 +1,31 @@
+package com.nytimes.android.external.store3.middleware.jackson
+
+import com.fasterxml.jackson.core.JsonFactory
+import com.fasterxml.jackson.databind.JavaType
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.kotlin.KotlinModule
+import com.nytimes.android.external.store3.base.Parser
+import com.nytimes.android.external.store3.util.ParserException
+import io.reactivex.annotations.NonNull
+import java.lang.reflect.Type
+import javax.inject.Inject
+
+class JacksonStringParser<Parsed> : Parser<String, Parsed> {
+
+    private val objectMapper: ObjectMapper
+    private val parsedType: JavaType
+
+    constructor(jsonFactory: JsonFactory, type: Type) {
+        objectMapper = ObjectMapper(jsonFactory).registerModule(KotlinModule())
+        parsedType = objectMapper.constructType(type)
+    }
+
+    @Inject
+    constructor(objectMapper: ObjectMapper, type: Type) {
+        this.objectMapper = objectMapper
```

Comment:
Same as before, either remove `this.` or add it on the other constructor as well.